Source/WebKit/UIProcess/Cocoa/MediaPermissionUtilities.mm

```
@@ -140,19 +140,19 @@ bool checkUsageDescriptionStringForSpeechRecognition()
 
     switch (reason) {
     case MediaPermissionReason::Camera:
-        return adoptNS([[NSString alloc] initWithFormat:WEB_UI_NSSTRING(@"Allow “%@” to use your camera?", @"Message for user camera access prompt"), visibleOrigin.get()]);
+        SUPPRESS_UNRETAINED_ARG return adoptNS([[NSString alloc] initWithFormat:WEB_UI_NSSTRING(@"Allow “%@” to use your camera?", @"Message for user camera access prompt"), visibleOrigin.get()]);
```

Why are these suppressions needed?

---

To silence errors like these:
- https://build.webkit.org/results/Apple-Sequoia-Safer-CPP-Checks/300671@main%20(5364)/scan-build-output/StaticAnalyzer/WebKit/StaticAnalyzerReports/report-MediaPermissionUtilities.mm-alertMessageText-11-205a0f.html#EndPath

Note that I have tried wrapping the calls to `WEB_UI_NSSTRING()` in a `RetainPtr { }.get()` but then we fail to build because the parameter to `initWithFormat` is no longer a string literal.

---

oh, ok. We should probably utilize `__attribute__((ns_returns_autoreleased))`